### ISSUE TITLE
The tree grid is unexpected working 

### ISSUE THREAD

**Issue by All-ForOne** [nhn/tui.grid#1233]
1. I initialized to all expanded tree grid and I collapsed tree rows.
2. Next I modified grid cell data.
3. All tree rows were collapsed but last child of tree is showing.

I expected collapsed tree rows were hide.

JSfiddle :  https://jsfiddle.net/nexd3c9g/1/

![__tree_bug](https://user-images.githubusercontent.com/73328076/100698010-c707db00-33da-11eb-925a-b4e00c1c9c12.gif)

**Comment by js87zz** [comment]
@All-ForOne 
I checked the bug it. We will fix it!

**Comment by js87zz** [comment]
The bug has been fixed in `v4.16.1`. And the CDN url has been changed in the past as below.
```js
// tui-grid => grid

// old url
<link rel="stylesheet" href="https://uicdn.toast.com/tui-grid/latest/tui-grid.css" />
<script src="https://uicdn.toast.com/tui-grid/latest/tui-grid.js"></script>

// current url
<link rel="stylesheet" href="https://uicdn.toast.com/grid/latest/tui-grid.css" />
<script src="https://uicdn.toast.com/tui-grid/latest/grid.js"></script>
```
